docs/getting-started/publishing-keys.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: default
+---
+
+# Key Publishing Guide
+
+This page explains how to configure the server and application to allow
+publishing keys.
+
+## Prerequisites
+
+If you haven't started developing a mobile app, please take a look at the
+[Android Exposure Notifications Reference App](https://github.com/google/exposure-notifications-android)
+
+You will need to
+[complete the necessary steps](https://support.google.com/googleplay/android-developer/contact/expo_notif_api)
+to use the exposure notifcations API. You should also ensure that you have
+taken the steps to comply with the
+[Terms of Service](https://blog.google/documents/72/Exposure_Notifications_Service_Additional_Terms.pdf).
+For instance, your app may not request any of the following permissions:
```

Comment:
I think we should drop this lest risk getting out of sync with future updates.

Reply:
dropped the outlined list.  Also moved to a shared doc component.